namespace-specific-policy/manual-rendering/scripts/render.sh

```diff
@@ -22,12 +22,8 @@ set -o errexit -o nounset -o pipefail
 REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
 cd "${REPO_ROOT}"
```

Comment:
nit: combine L23 and L25 into one command.

Reply:
Done